src/ui/pages/NotificationDetails/components/CredentialRequest/CredentialRequest.types.ts

```diff
@@ -6,15 +6,15 @@ interface CredentialRequestProps {
   pageId: string;
   activeStatus: boolean;
   notificationDetails: KeriaNotification;
-  credentialRequest: CredentialsMatchingApply;
+  credentialRequest: Omit<CredentialsMatchingApply, "identifierId">;
```

Comment:
Why omit here? It looks like we are passing a full `CredentialsMatchingApply` to `CredentialRequestInformation` etc